src/entities/offerings.ts

```diff
@@ -408,7 +408,18 @@ export interface Offering {
    */
   readonly weekly: Package | null;
 
+  /**
+   * The paywall components configured in the RevenueCat dashboard, if available.
+   * @internal
+   */
   readonly paywall_components: PaywallData | null;
+
+  /**
+   * The UI configuration needed to render the paywall components, usually set
+   * together
+   * @internal
+   */
+  readonly ui_config?: UIConfig;
```

Comment:
Would it be worth changing these to camelCase now that they are internal? Can also happen in future PRs though.